tests/staking/tier-threshold.test.ts

```diff
@@ -0,0 +1,180 @@
+import { expect } from 'vitest'
+import { given } from '../../helpers'
+
+given('astar')('Number of slots adjusted based on price', async ({ networks: { astar } }) => {
+  const advanceNextEra = async () => {
+    const state = await astar.api.query.dappStaking.activeProtocolState<any>()
+    await astar.dev.newBlock({ count: 1, unsafeBlockHeight: state.nextEraStart.toNumber() })
+  }
+
+  // set price to $0.10
+  await astar.dev.setStorage({
+    priceAggregator: {
+      valuesCircularBuffer: {
+        head: 1,
+        buffer: [
+          100_000_000_000_000_000n, // $0.10
+        ],
+      },
+    },
+  })
+
+  await advanceNextEra()
+
+  expect((await astar.api.query.dappStaking.tierConfig()).toHuman()).toMatchInlineSnapshot(`
+      {
+        "numberOfSlots": "150",
+        "rewardPortion": [
+          "25.00%",
+          "47.00%",
+          "25.00%",
+          "3.00%",
+        ],
+        "slotsPerTier": [
+          "7",
+          "30",
+          "45",
+          "67",
+        ],
+        "tierThresholds": [
+          {
+            "DynamicTvlAmount": {
+              "amount": "306,666,666,666,666,666,901,705,287",
```

Comment:
These numbers need to be updated, they're from the old faulty runtime.

Reply:
running CI will catch it but it's blocked because of this https://github.com/AstarNetwork/e2e-tests/pull/32

Reply:
@Dinonard updated